### PR TITLE
Fix: remove build errors

### PR DESCRIPTION
**Pull request by atomictangerine**

### CODE REVIEW

File: pages/_document.js [hunk 1]
```diff
@@ -16,8 +16,13 @@ const getCssAndReset = () => {
 
 export default class Document extends NextDocument {
   static async getInitialProps(ctx) {
-    const initialProps = await NextDocument.getInitialProps(ctx);
-    return { ...initialProps };
+    let initialProps = { html: "<></>" };
```
Comment (atomictangerine) [inline pages/_document.js:19]: is expecting to receive initialProps for components that we have doc pages for even though they haven't been built yet.

File: services/index.js [hunk 2]
```diff
@@ -6,3 +6,5 @@ export * from "./getNavigation";
 export * from "./getPropsTable";
 export * from "./getDependencies";
 export * from "./getPackageData";
+
+export const exemptList = ["image", "text"];
```
Comment (atomictangerine) [inline services/index.js]: Lists the components that we haven't built yet but that have some doc pages. This was the easiest way to check them, without reading files.

**Review by artmsilva — APPROVED** [review]

**Review by artmsilva — COMMENTED** [review]
We could have conditionally called the getPackageData service fn by using the coming soon label in getStaticProps.

**Comment by atomictangerine** [comment]
I did try that. However, we still need to get an initialProps object as well as a couple of other objects in order for the pages to prerender correctly.

File: pages/_document.js (hunk 1 above)
Comment (artmsilva) [reply pages/_document.js:19]: Great!

**Comment by artmsilva** [comment]
> I did try that. However, we still need to get an initialProps object as well as a couple of other objects in order for the pages to prerender correctly.

I agree. Let's not remove that.

**Review by clairehhelms — APPROVED** [review]

**Pull request merged**